src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -235,9 +236,19 @@ public void canCreateUpdateAndDeleteFares() throws IOException, SQLException, In
         assertThat(createdFare.fare_id, equalTo(fareId));
         assertThat(createdFare.fare_rules[0].fare_id, equalTo(fareId));
 
+        // Ensure transfers value is null.
```

Comment:
Can you expand this comment a bit to indicate that this part of the test is checking the database integrity?